lib/deploy/functions/release.js

```diff
@@ -110,123 +117,53 @@ module.exports = function(context, options, payload) {
     }
   }
 
-  function _reportResults(successfulCalls, failedCalls) {
-    function logFailedOps(ops) {
-      _.forEach(ops, function(operation) {
-        var parts = operation.func.split('/');
-        var functionName = parts[5];
-        utils.logWarning(chalk.bold.yellow('functions[' + functionName + ']: ') + 'Deploy Error: ' + operation.error.message);
-      });
+  function _pollAndManageOperations(operations) {
+    var interval;
+    // Poll less frequently when there are many operations to avoid hitting read quota.
+    if (_.size(operations) > 15) {
```

Comment:
This will still eventually blow through quotas, right? At what size do we blow through quotas ever time?